Source/BugsnagLogger.h

```diff
@@ -1,12 +1,38 @@
+/**
+ * Higher-level user-accessible BugsnaglLogging configuration.  Controls how
+ * verbose the internal Bugsnag logging is.  Not related to logging Events or
+ * other errors with the Bugsnag server.
+ *
+ * Users can configure a custom logging level in their app as follows:
+ *
+ * When using Cocoapods to install Bugsnag you can add a `post-install` section
+ * to the Podfile:
+ *
+ *     post_install do |rep|
+ *         rep.pods_project.targets.each do |target|
+ *             if target.name == "Bugsnag"
+ *                 target.build_configurations.each do |config|
+ *                     config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', 'BSG_LOG_LEVEL=BSG_LOGLEVEL_INFO']
+ *                 end
+ *             end
+ *         end
+ *     end
+ *
+ * Change the value of `BSG_LOG_LEVEL` to one of the levels given below and run `pod install`.
```

Comment:
Is it possible to alter the log level outside of using Cocoapods? If so it'd be good to document how to achieve this in a package-manager agnostic way.

Reply:
Agreed, as noted in the PR. An internal ticket has been raised to address this. While the mechanism (`BSG_LOG_LEVEL`) __is__ applicable across installation methods (Cocoapods, Carthage and Manually), documenting the specifics for all methods of library integration is left for a later date.

Reply:
As long as a separate task has been raised and it will be addressed as part of the Notifier Spec project I'm happy to leave this for now.